heudiconv/convert.py

```diff
@@ -975,9 +979,17 @@ def save_converted_files(
         is_uncombined = (
             len(set(filter(bool, channel_names))) > 1
         )  # Check for uncombined data
-        is_complex = (
-            "M" in image_types and "P" in image_types
-        )  # Determine if data are complex (magnitude + phase)
+        CPLX_PARTS = ["MAGNITUDE", "PHASE", "IMAGINARY", "REAL"]
```

Comment:
I need to "turn off" for today -- logic escapes me on how to possibly  reuse that `type_to_part` here ;) may be nohow ;)

Reply:
thanks a lot for the feedback, will try to wrap my head around that.